frontend/src/config/types.ts

```diff
@@ -361,17 +361,17 @@ export class BoundaryLayerProps extends CommonLayerProps {
   isPrimary?: boolean | undefined;
 }
 
-export enum LabelType {
+export enum DataType {
   Date = 'date',
   Text = 'text',
   Number = 'number',
-  ValuesLabels = 'values_labels',
+  LabelMapping = 'labelMapping',
 }
 
 interface FeatureInfoProps {
```

Comment:
Looks like we have multiple `FeatureInfoProps` interfaces defined. Is this expected?